package.json

```diff
@@ -64,17 +64,17 @@
     "tinymce": "5.8.1"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.28.3",
-    "@typescript-eslint/parser": "^4.28.3",
-    "eslint": "^7.30.0",
+    "@typescript-eslint/eslint-plugin": "^5.5.0",
+    "@typescript-eslint/parser": "^5.5.0",
+    "eslint": "^8.4.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-tsdoc": "^0.2.14",
-    "husky": "^7.0.1",
-    "lint-staged": "^11.0.1",
-    "prettier": "^2.3.2",
-    "tsd": "^0.17.0",
-    "typescript": "^4.4.3"
+    "husky": "^7.0.4",
+    "lint-staged": "^12.1.2",
+    "prettier": "^2.5.1",
+    "tsd": "^0.19.0",
```

Comment:
We should not upgrade to this version yet, it includes a forked typescript of 4.5, that also had those performance issues. At least in my system this forked version was used and the performance regression was only gone after I downgraded both `tsd` and `typescript`.

Reply:
Yeah, the pipelines seem to take a couple of seconds longer. So I’ll downgrade to 0.18.0

Reply:
done